tools/keytools/Makefile

```diff
@@ -162,6 +162,18 @@ OBJS_REAL+=\
 	$(XMSSDIR)/utils.o
 endif
 
+# Add wolfcrypt ML-DSA (dilithium) implementation.
+ifeq ($(SIGN),ML_DSA)
+  OBJS_REAL+=$(WOLFDIR)/wolfcrypt/src/dilithium.o
+
+  CFLAGS += -D"WOLFSSL_DILITHIUM_FIPS204_DRAFT" \
```

Comment:
I would prefer these build options reside in tools/keytools/user_settings.h. Please move this and others if possible.

Reply:
I'll move these, agree will be tidier